style.scss

```diff
@@ -29,9 +28,15 @@ body {
     font-weight: bold;
 }
 
+a {
+    text-decoration: none !important;
```

Comment:
Right decision, and let's refine it just a bit:
1. Since it's a rule for all links please move it higher, right under the rule for `body` and `html`.
2. Don's use `!important`, it's redundant here.
3. Rule for `border` is not needed, because it's not set by default. So remove it, please.